frontend/tests/service/utils.ts

```diff
@@ -0,0 +1,6 @@
+export const SERVICE_TABLE_HEADERS = {
```

Comment:
no need already there is GraphTitle let's use that

Reply:
checking